src/scripts/fetcher/TachiyomiAZ.js

```diff
@@ -59,19 +59,21 @@ export class TachiyomiAZ extends BaseVariant {
 				this.store.commit("setRelease", {
 					variant: this.variant,
 					type: type,
-					newData: stripPreviewData(data),
+					newData: this.stripPreviewData(data),
 				});
 				resolve(this.fetchLocal(type));
 			});
-			resolve(this.fetchLocal(type));
 		});
 	};
 
 	stripPreviewData = data => {
+		var versionRelease = new Date(data * 1000).toLocaleDateString("en-US");
```

Comment:
Probably just use user default locale instead, otherwise we will confuse the Europeans 
```suggestion
		var versionRelease = new Date(data * 1000).toLocaleDateString();
```

Reply:
oops I didn't even realize, I copy and pasted that 🤣